src/umpire/strategy/QuickPool.cpp

```diff
@@ -207,7 +216,22 @@ void QuickPool::release()
 
       UMPIRE_POISON_MEMORY_REGION(m_allocator, chunk->data, chunk->chunk_size);
       m_actual_bytes -= chunk->chunk_size;
-      aligned_deallocate(chunk->data);
+
+      try {
+        aligned_deallocate(chunk->data);
+      }
+      catch (...) {
+        if (m_is_destructing) {
+          //
+          // Ignore the error if we are destructing as the resource to which we
+          // are releasing to may have gone away
+          //
+          UMPIRE_LOG(Error, "Pool is destructing, Exception Ignored");
```

Comment:
Fix comment, log the exception message.

Reply:
Are you asking me to distinguish between an Umpire exception and some other type of exception?  If it is an Umpire exception, the message will be logged by UMPIRE_ERROR.

I don't understand what you are looking for in the comment.  Please suggest an alternative?

Reply:
Done